package.json

```diff
@@ -36,13 +37,15 @@
     "start-built-server": "./scripts/typescript/start_built_server.sh",
     "start-tests-server": "./scripts/tests/api/start_tests_server.sh",
     "test": "./scripts/tests/run_tests.sh",
+    "test-federated": "export INV_FEDERATED_TESTS=true ; ./scripts/tests/run_tests.sh",
```

Comment:
Should we skip all tests in `tests/integration/data` when running this script `npm run test-federated`, as it concerns entities mainly ?

Reply:
Done in 4484c9681